frontend/src/Context/index.tsx

```diff
@@ -7,6 +7,8 @@ interface QuickstartState {
   accessToken: string | null;
   itemId: string | null;
   isError: boolean;
+  paymentInitiation: boolean | null;
+  backend: boolean;
```

Comment:
we have many different keys here used to determine state and errors, such as `linkSuccess`, `isItemAccess`, `backend`, `linkToken == null`, etc. Maybe we could simplify this a bit in the future, i have some ideas!

Reply:
okay